sklearn/linear_model/_coordinate_descent.py

```diff
@@ -859,6 +859,10 @@ class ElasticNet(MultiOutputMixin, RegressorMixin, LinearModel):
     If so, then additionally check whether the dual gap is smaller than `tol` times
     :math:`||y||_2^2 / n_{\text{samples}}`.
 
+    For an example, see
+    :ref:`examples/linear_model/plot_elastic_net_precomputed_gram_matrix_with_weighted_samples.py
+    <sphx_glr_auto_examples_linear_model_plot_elastic_net_precomputed_gram_matrix_with_weighted_samples.py>`.
+
```

Comment:
This should go to the docstring of `precompute` in this docstring.

Reply:
I shall fix it.

Reply:
fixed now.